cdap-proto/src/main/java/co/cask/cdap/proto/id/EntityId.java

```diff
@@ -85,10 +85,18 @@ protected EntityId(EntityType entity) {
 
   protected abstract Iterable<String> toIdParts();
 
+  /**
+   *  Deprecated method to rename to getEntityType(). Use {@link #getEntityType()}
+   */
+  @Deprecated
   public final EntityType getEntity() {
     return entity;
   }
 
+  public final EntityType getEntityType() {
```

Comment:
should we name the method as `EntityId.getType` instead of `EntityId.getEntityType`?